lib/tasks/fix_xccdf_reports.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+desc <<-END_DESC
+END_DESC
+
+task fix_xccdf_reports: :environment do
+  noop = ENV['NOOP']
+  start_time = Time.now.utc
+  puts '---------- NOOP --------' if noop
+  puts "Starting fix_xccdf_reports job at #{start_time}"
+
+  MigrateXCCDFReportsJob.perform_later(ENV['ACCOUNT_NUMBER'], noop)
```

Comment:
Maybe we should also create another rake task which calls this one with all accounts currently in the database.

Reply:
:+1: